WordPress/wordpress-playground
# bootPlaygroundRemote: Defer setting the iframe src initially, to prevent flash of unstyled content

## What?

Removes a line in `bootPlaygroundRemote` that initially sets iframe src to route `/`.

This pull request defers setting the `src` attribute, letting the client set it when the website is ready, e.g., afer blueprint steps are run and desired theme activated.

Possibly this could be an opt-in behavior.

## Why?

Prevents flash of unstyled content, for example, when a default theme is shown before switching to a newly installed theme.

## How?

Should check if this doesn't break something unexpected. The PR will require all callers of `bootPlaygroundRemote` to set the iframe src themselves. Otherwise it will stay empty.

## Testing Instructions


1. Check out the branch.
2. Run `npm run dev`.
3. Visit a URL with a different theme activated, and confirm that the default theme does not show for a moment before switching to the active theme.
